tests/h/views/api/groups_test.py

```diff
@@ -184,6 +186,62 @@ def pyramid_request(self, pyramid_request, factories):
         return pyramid_request
 
 
+@pytest.mark.usefixtures('group_service',
+                         'user_service',
+                         'request_auth_client',
+                         'validate_auth_client_authority')
+class TestAddMember(object):
+
+    def test_it_adds_user_from_request_params_to_group(self,
+                                                       factories,
+                                                       pyramid_request,
+                                                       group_service,
+                                                       user_service):
+        group = factories.Group(authority='example.com')
+        user = factories.User(authority='example.com')
+
+        user_service.fetch.return_value = user
+
+        pyramid_request.matchdict['userid'] = user.userid
+        pyramid_request.matchdict['pubid'] = group.pubid
+
+        resp = views.add_member(group, pyramid_request)
+
+        group_service.member_join.assert_called_once_with(group, user.userid)
+        assert isinstance(resp, HTTPNoContent)
+
+    def test_it_fails_with_mismatched_user_and_group_authorities(self,
```

Comment:
Might we rename this test to indicate in what manner it fails (hint: the HTTP response)?